api/go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/hashicorp/go-secure-stdlib/parseutil v0.1.6
 	github.com/hashicorp/go-secure-stdlib/strutil v0.1.2
 	github.com/hashicorp/hcl v1.0.0
+	github.com/hashicorp/vault/sdk v0.9.0
```

Comment:
I don't see why api would need to change its dependencies for your PR, and also this is undoing some of my recent work that ensured that api doesn't depend on sdk; can you either revert this change or explain to me why it's necessary please?

Reply:
Surely unnecessary. I think this was the result of an intermediate step.
I removed it.

Reply:
I additionally also executed go mod tidy.